application/repositories.go

```diff
@@ -33,3 +34,30 @@ type SearchRepository interface {
 type IdentityRepository interface {
 	List(ctx context.Context) (*app.IdentityArray, error)
 }
+
+// WorkItemLinkCategoryRepository encapsulates storage & retrieval of work item link categories
+type WorkItemLinkCategoryRepository interface {
```

Comment:
No need to add the interfaces here. They are already defined in:
* `workitem/link/link-repository.go`
* `workitem/link/category-repository.go`
* `workitem/link/type-repository.go`